### PR TITLE
Put text about "trust" earlier and simplify the security consideration

### PR DESCRIPTION
**Pull request by paulehoffman**

### CODE REVIEW

File: draft-ietf-doh-dns-over-https-latest.mkd [hunk 1]
```diff
@@ -383,17 +383,29 @@ HTTP/2 {{RFC7540}}. Earlier versions of HTTP are capable of conveying
...
+query. In other words, a client MUST only trust a DNS API server
+that is configured as trustworthy. A client MUST NOT trust a DNS API 
+server simply because it was discovered, or
+because the client was told to trust the DNS API server by an
+untrusted party. 
+
+This specification does not extend DNS resolution privileges 
+to URIs that are not recognized by the client as trusted DNS API servers. 
+As such, use of untrusted servers is out of scope of this document 
+(however Section 8 discusses some of the security risks of using them).
```
Comment (martinthomson) [inline draft-ietf-doh-dns-over-https-latest.mkd]: Don't use bare section references.

Comment (paulehoffman) [reply draft-ietf-doh-dns-over-https-latest.mkd]: Fixed

File: draft-ietf-doh-dns-over-https-latest.mkd [hunk 2]
```diff
@@ -504,15 +516,13 @@ than the security implications of HTTP caching for other protocols
...
-answer those queries (through its DNS API service). An untrusted
-DNS API server can thus easily cause damage by poisoning a client's
-cache with names that the DNS API server chooses to poison. A
-client MUST NOT trust a DNS API server simply because it was
-discovered, or because the client was told to trust the DNS API
-server by an untrusted party. Instead, a client MUST only trust DNS
-API server that is configured as trustworthy.
+server is in a position to choose which DNS names it forces a client
+to resolve (through its web service) and also be the one to answer
+those queries (through its DNS API service).
```
[the remark this answers is not in the document]
Comment (paulehoffman) [reply draft-ietf-doh-dns-over-https-latest.mkd]: Possibly fixed

**Comment by paulehoffman** [comment]
Closing this one because #157 was accepted. It focuses more on authorization than trust.

**Pull request closed without merging**